### ISSUE TITLE
The report makes it seem like the mutations are performed on the test project

### ISSUE THREAD

**Issue by rouke-broersma** [Faultify/Faultify#20]
![image](https://user-images.githubusercontent.com/3595932/107123686-326c9e00-689f-11eb-9e31-4a62db76180a.png)

I would expect my source code to be listed under the dll or project name where the source code lives. Now I have to 'click through' on the test project binary name to get to the source code. At first I thought faultify had accidentally mutated my test project instead of my source project.

**Comment by TimonPost** [comment]
Good remark. I think it's a good idea to have the mutated assembly name here instead. 

I tried to find a reasonable format, in the past, we had the following structure:

- test assembly
     - test name
         - arithmetic mutators
             - mutations

However, we experienced some problems like the HTML-file size which grew into big sizes. I think a complete restructure of the report is mandatory in order for it to be effective and or usefull. I would like to see things like: Classnames, filenames, mutation categories, test names etc.. Perhaps, in relation to #14, we can maintain the styker json structure to make it compatible with its dashboard. That's something for another issue.
